android/buildSrc/build.gradle.kts

```diff
@@ -0,0 +1,7 @@
+repositories {
+    jcenter()
+}
+
+plugins {
+    `kotlin-dsl`
+}
```

Comment:
the `buildSrc` seems to be an experimental feature? I actually like it but I am not sure if it messes up things at the moment.

Will try that again next week during the stream and if possible replace the `rootProject.extra...` parts with the `Versions.kt` again

Reply:
I also don't like the buildSrc thing probably I will replace it